alternator/executor.cc

```diff
@@ -2800,7 +2832,7 @@ future<executor::request_return_type> executor::batch_write_item(client_state& c
                 }
                 used_keys.insert(std::move(mut_key));
             } else if (r_name == "DeleteRequest") {
-                const rjson::value& key = (r->value)["Key"];
+                const rjson::value& key = get_member_object(r.value, "Key", "DeleteRequest");
```

Comment:
Here too, I guess the existing code already verifies that the "Key" has the right members and complains if it doesn't, so we don't need to check in advance that the object isn't empty. We should check the member exists, and is an object - definitely - but not that it's not empty.